solr/contrib/ltr/example/solrconfig.xml

```diff
@@ -876,11 +875,7 @@
        <str name="indent">true</str>
        <str name="df">text</str>
      </lst>
-<arr name="last-components">
-      <!-- Use the component in your requestHandler -->
-      <str>ltrComponent</str>
-    </arr>
-  </requestHandler>
+ </requestHandler>
```

Comment:
there's the example svm training python script as well which might have been missed